### ISSUE TITLE
Support 'erlang:list_to_integer/1'

### ISSUE THREAD

**Issue by mcherep** [aggelgian/cuter#80]
Here's an example.

```erlang
-spec average(A :: string(), B :: string()) -> number().
average(A, B) ->
  A_Int = list_to_integer(A),
  B_Int = list_to_integer(B),
  A_Int+B_Int/2.
```

**Comment by kostis** [comment]
Hola @mcherep !

Thanks for the prompt to extend the support for BIFs, but allow me a comment.  Your example program is not as it should be.  We want support for Erlang BIFs that allow CutEr to find all errors in programs, not only the not so interesting ones.  So, it's better that the example program clearly shows a point where the bug(s) may be.  To see what I mean, the following program involving `list_to_integer/1` is more interesting:
```erlang
-spec l2i(string()) -> ok.
l2i(L) ->
  case list_to_integer(L) of
    42 -> error(bug);
    I when is_integer(I) -> ok
  end.
```
It contains a more interesting case of an error that proper support for `list_to_integer/1` should allow CutEr to find.

**Comment by mcherep** [comment]
Thanks. I copied and pasted, and forgot the whole example!

**Comment by aggelgian** [comment]
Bug fixed by #83.
